tests/test_core.py

```diff
@@ -6057,6 +6057,17 @@ def run():
     self.build(open(path_from_root('tests', 'test_wasm_intrinsics_simd.c')).read(),
                self.get_dir(), os.path.join(self.get_dir(), 'src.cpp'))
 
+  # Tests invoking the NEON SIMD API via arm_neon.h header 
+  @wasm_simd
+  def test_neon_wasm_simd(self):
+    # Improves test readability
```

Comment:
Agree, but I'm not sure including a comment about it is useful.

Reply:
Mmm, I don't remember why this is here. I'll remove it.